docs/tutorials/deploy_openfaas_workload.md

```diff
@@ -0,0 +1,291 @@
+# OpenFaaS Extended Workload
+
+## Install
+
+In this tutorial, we assume you have already successfully installed Rudr. If not, you could see the [Installation Guide](../setup/install.md) to get help.
+
+### Install OpenFaaS Operator with Helm v3
+
+After Rudr was installed, we need to install [OpenFaaS Operator](https://github.com/openfaas-incubator/openfaas-operator), that will be the real workload executor.
+
+```shell script
```

Comment:
We recommend downstream use our CLI to prevent overhead and maintenance issues.

k3sup app install openfaas --helm3

☝️

You'll also want the flag for the operator so run --help to find out.

Reply:
Does k3sup work with any K8s distribution?

Reply:
I think this work could be done in another PR, including how to install `k3sup`.

Reply:
Yes indeed it does and it should be in this PR

Reply:
so many people blindly copy and paste the very detailed installation instructions for the project and then we change them there is too much churn

Reply:
OK，I'll change it

Reply:
By the way, I copy/paste the instructions here: https://github.com/openfaas-incubator/openfaas-operator#deploy-openfaas-with-the-operator, it didn't use k3sup.

Reply:
As I tried, Using `k3sup` install openfaas is a little hard, as network connecting with github in China is a little slow, so it always breakdown.

1. `curl -sLS https://get.k3sup.dev | sh` this command didn't use CDN, it directly download from GitHub release which make it even impossible to download. So I download them directly from release page of GitHub .
2. `k3sup app install openfaas --helm3` this command didn't use my local helm, instead, it download helm 3.1, which need network and breakdown again.

So I hope to keep the instructions as it is in this PR.